src/Console/ConsoleApplication.php

```diff
@@ -43,7 +43,7 @@ public function doRun(InputInterface $input, OutputInterface $output): int
     {
         // @fixes https://github.com/rectorphp/rector/issues/2205
         $isXdebugAllowed = $input->hasParameterOption('--xdebug');
-        if (! $isXdebugAllowed) {
+        if ($isXdebugAllowed) {
```

Comment:
by register option xdebug to `ProcessConfigureDecorator`, the is xdebug has parameter option seems can use truthy check 👍